testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py

```diff
@@ -376,11 +374,10 @@ def params_from_base_suite_setup(request):
         # Create primary index
         password = "password"
         log_info("Connecting to {}/{} with password {}".format(cbs_ip, enable_sample_bucket, password))
-        sdk_client = Bucket('couchbase://{}/{}'.format(cbs_ip, enable_sample_bucket), password=password, timeout=SDK_TIMEOUT)
+        sdk_client = Bucket('couchbase://{}/{}'.format(cbs_ip, enable_sample_bucket), password=password)
```

Comment:
timeout is removed, Can you addd the timeout , otherwise it might fail inconsistently

Reply:
it is not supported